src/pybind/mgr/dashboard/frontend/src/app/shared/components/form-advanced-fieldset/form-advanced-fieldset.component.scss

```diff
@@ -0,0 +1,22 @@
+/*
+  Overrides the active and focus states of bootstrap accordion,
+  in order to align the accordion css with rest of the form.
+
+  Summary of overrides:
+  - font-size was 1 rem
+  - color was blue when higlighted
+  - border,outlines and box-shadow were set
+  - accordion down button in active form was a blue svg icon
+
+ */
+.cd-form-advanced-fieldset-buttton {
+  --bs-accordion-btn-active-icon: var(--bs-accordion-btn-icon) !important;
+  background-color: inherit !important;
+  border: hidden !important;
+  border-color: inherit !important;
+  box-shadow: none !important;
+  color: inherit !important;
+  font-size: inherit;
+  outline: 0 !important;
+  text-decoration: none !important;
+}
```

Comment:
is the `!important` tag necessary? Usually its not a good practice to mark something as `!important` in the child css components. This makes it impossible if we want to override these styles in the future from the parent css file, especially since we override our styles through css in the downstream.

Reply:
Its all bootstrap css..so bootstrap has "accordion" css which is basically putting these.
The look and feel is very different and everything is blue like you see in add osd wizard form.

Let me see if I could do something here..

Reply:
It worked without `important` , thanks!